README.md

```diff
@@ -144,6 +144,9 @@ Installing a snap is very quick. Snaps are secure. They are isolated with all of
         # or in background
         docker run -it -d -v /monero/chain:/root/.bitmonero -v /monero/wallet:/wallet -p 18080:18080 monero
 
+
+**Warning** make sure to use `tty: true` with `docker-compose`, if `monerod` don't have a tty it will exit once it's initialized.
```

Comment:
I think if you set the flag `--non-interactive` this may not be needed. 

#1763 was an issue about tty being required for containers and it was addressed via #1897 which added that flag.

I don't use a container setup but based on those comments I think that may be the case. Ignore this if I'm incorrect though just was looking into interactive mode the other day so ran into that issue.

Reply:
fixed https://github.com/monero-project/monero/pull/2835/commits/81ac0e91cb7302461020eacd1ba5103ceb10cf4a